core/src/components/button/button.ios.vars.scss

```diff
@@ -34,7 +34,7 @@ $button-ios-height:                                   3.1em !default;
 $button-ios-border-radius:                            14px !default;
 
 /// @prop - Font size of the button text
-$button-ios-font-size:                                16px !default;
+$button-ios-font-size:                                min(1rem, 22px) !default;
```

Comment:
I couldn't find a native example of a solid button to compare against but I did find a clear one and since that was 1px larger before the conversion I just decreased the max size by `1px` compared to clear:

```scss
$button-ios-clear-font-size:                          min(1.0625rem, 23px) !default;
```

Reply:
Thank you! I haven't even been testing going up to 310%...I need to take another look at these values.